src/getBuildingId.py

```diff
@@ -32,9 +31,10 @@ def getBuildingId(event, context):
                 'headers': {'Content-Type': 'application/json'},
                 'body': json.dumps({'error': 'Building not found'})
             }
-    """Error if path not specified"""
-    return {
-       'statusCode': 404,
-       'headers': {'Content-Type': 'application/json'},
-       'body': json.dumps({'error': 'Path not found'})
-    }
+    else:
+        """No path parameters"""	
+        return {
+		    'statusCode': 400,
+		    'headers': {'Content-Type': 'application/json'},
+		    'body': json.dumps({'error': 'Path not found'})
+	    }
```

Comment:
Just checked this out, turns out you're mixing tabs and spaces here. So while it looks fine when tabs are configured to 4 spaces, when it's rendered somewhere with a different setting it looks bleh. Fix that. The rest of the file is using spaces. Your editor should have an option to convert them automatically, I think?